src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,43 @@
+.menu {
+
+  background-color: $secondary-color;
+  height: 100vh;
+
+  &__icon {
+    @include top-actions;
+    margin-bottom: 40px;
+
+    @include Tablet {
+      margin-bottom: 100px;
+    }
+  }
+
+  &__list {
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 18px;
```

Comment:
It seems like the Sharp Sans No2 Medium and Avenir fonts are not being used. Make sure to apply these fonts to the text as per the task description.